pytse_client/utils/symbols.py

```diff
@@ -6,6 +6,6 @@
 
 def map_index_to_symbols():
     symbols_file = f"{config.pytse_dir}/data/symbols_name.json"
-    with open(symbols_file, "r") as json_file:
+    with open(symbols_file, "r", encoding="utf8") as json_file:
```

Comment:
بنظر میرسه این باید "utf-8" باشه مشکلی نداره اینطوری نوشته بشه؟ من داشتم مثال‌های پایتون رو میخوندم دیدم
https://docs.python.org/3/howto/unicode.html

Reply:
تفاوتی ندارد. اصلیش utf_8 است بقیه کمکی هستند. باز هم اگر فکر می کنید دش داشته باشد بهتر است بگید اصلاح کنم.

https://docs.python.org/3.8/library/codecs.html

Standard Encodings
Python comes with a number of codecs built-in, either implemented as C functions or with dictionaries as mapping tables. The following table lists the codecs by name, together with a few common aliases, and the languages for which the encoding is likely used. Neither the list of aliases nor the list of languages is meant to be exhaustive. Notice that spelling alternatives that only differ in case or use a hyphen instead of an underscore are also valid aliases; therefore, e.g. 'utf-8' is a valid alias for the 'utf_8' codec.

CPython implementation detail: Some common encodings can bypass the codecs lookup machinery to improve performance. These optimization opportunities are only recognized by CPython for a limited set of (case insensitive) aliases: utf-8, utf8, latin-1, latin1, iso-8859-1, iso8859-1, mbcs (Windows only), ascii, us-ascii, utf-16, utf16, utf-32, utf32, and the same using underscores instead of dashes. Using alternative aliases for these encodings may result in slower execution.

Reply:
داخل برنامه هم چند جای دیگه از همین کدینگ استفاده شده است. اگر قرار به تغییر شد همه را اصلاح کنم.

Reply:
دو جا هم از open استفاده شده است بدون انکودینگ که اونها هم بهتر است اصلاح شوند. به نظرم بهتر است این برنچ را ببندیم همه موارد را در یک برنچ دیگه اصلاح کنیم.

Reply:
آها مرسی که گفتی. اگر میخوای همرو با یک مرج ریکوئست انجام بدی یکی دیگه باز کن این رو هم میتونی ببندی اگر نه اینو مرج کنم.

Reply:
بزار همه را پس همینجا اعمال کنیم فقط اینکه دش بزاریم یا نه؟!

Reply:
همرو همینجا اعمال کنیم. دش هم لازم نیست همین اوکیه.